bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/model/BazelModel.java

```diff
@@ -117,4 +117,9 @@ public BazelModelManager getModelManager() {
     public int hashCode() {
         return BazelModel.class.hashCode();
     }
+
+    public void invalidateAll() throws CoreException {
+        getInfoCache().invalidateAll();
+        getInfo().invalidate();
```

Comment:
That shouldn't be needed. The model info should also be in the cache. Is it not getting evicted?